external/Makefile.am

```diff
@@ -7,3 +13,15 @@ endif
 # This is intentionally left empty. This will dev/null any attempt
 # to do distcheck or distirbute anything in external
 DIST_SUBDIRS=
+
+# If you want to just skip testing, but still package the 
+# subdir, add it here
```

Comment:
I would argue this one is needed. The dist-hook target is pretty generic, so explaining what this one is for I think is worthwhile.